fix(waf): fix WAF reference table resource lint error and doucument description (#5406)
huaweicloud/terraform-provider-huaweicloud: docs/resources/waf_reference_table.md

The following arguments are supported:

* `region` - (Optional, String, ForceNew) The region in which to create the WAF reference table resource.

>   If omitted, the provider-level region will be used. Changing this setting will push a new reference table.

Reviewer: ```suggestion
  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
```

* `name` - (Required, String) Specifies the name of the reference table. Only letters, digits, hyphens (-),

>   underscores(_) and dots(.) are allowed. The maximum length is 64 characters.

Reviewer: ```suggestion
  underscores(_) and dots(.) are allowed. The maximum length is `64` characters.
```

* `type` - (Required, String, ForceNew) Specifies the type of the reference table.
  The valid values are **url**, **user-agent**, **ip**, **params**, **cookie**, **referer** and **header**.

>   Changing this setting will push a new reference table.

Reviewer: ditto

* `conditions` - (Required, List) Specifies the conditions of the reference table.

* `description` - (Optional, String) Specifies the description of the reference table.

>   The maximum length is 128 characters.

Reviewer: ```suggestion
  The maximum length is `128` characters.
```
